src/subscriptions/subscriptionsReducer.js

```diff
@@ -15,11 +16,6 @@ import { filterArray } from '../utils/immutability';
 
 const initialState: SubscriptionsState = NULL_ARRAY;
 
-const updateSubscription = (state, event) =>
-  state.map(sub =>
-    sub.stream_id === event.stream_id ? { ...sub, [event.property]: event.value } : sub,
-  );
```

Comment:
This inlining (to the two call sites) can happen as its own small NFC prep commit.

That way it's clear that the case of a subscription-update event doesn't change, and it stands out more what changes in the case of a stream-update event.